src/settings.plugin.js

```diff
@@ -259,6 +249,125 @@ wikiaJWPlayerSettingsPlugin.prototype.updateCurrentQuality = function (data) {
 	}
 };
 
+wikiaJWPlayerSettingsPlugin.prototype.addCaptionListener = function () {
+	var clickHandler = this.captionsClickHandler.bind(this);
+
+	this.player.on('captionsList', function (event) {
+		// tracks always include "off" item
+		if (event.tracks.length > 1) {
+			this.currentlySelectedCaptions = this.getSuitableCaptionsIndex(
+				this.captionLangMap[this.getUserLang()],
+				event.tracks
+			);
+
+			this.getLabelElement(this.captionsToggle)
+				.addEventListener('click', clickHandler);
+
+			this.wikiaSettingsElement.classList.remove('are-captions-empty');
```

Comment:
How about extracting `'are-captions-empty'` to variable as it's used twice

Reply:
will be fixed in captions-list PR